homeworks/09-node-chokidar/app.ts

```diff
@@ -0,0 +1,24 @@
+import { EventEmitter } from 'events';
+
+import { DirWatcher, DirWatcherConfig } from './modules/dirWatcher';
+import { Importer, IMPORTED_EVENT } from './modules/importer';
+
+const PATH_TO_DATA = './data';
+
+const DIR_WATCHER_CONFIG: DirWatcherConfig = {
+  dirPath: PATH_TO_DATA,
+  fileExtention: 'CSV',
+  watchInterval: 10000,
```

Comment:
```
watchInterval: process.env.INTERVAL || 10000,
```
i do think that it'd be nice to have possibility to set an interval via env arguments.

Reply:
added in https://github.com/vbarbasins/evo-ts-bootcamp/pull/10/commits/9e1253a8620799f26e2af37c0a2741153355ecd4